helix-term/src/ui/editor.rs

```diff
@@ -1521,7 +1527,14 @@ impl Component for EditorView {
     fn cursor(&self, _area: Rect, editor: &Editor) -> (Option<Position>, CursorKind) {
         match editor.cursor() {
             // All block cursors are drawn manually
-            (pos, CursorKind::Block) => (pos, CursorKind::Hidden),
+            (pos, CursorKind::Block) => {
+                if self.terminal_focused {
+                    (pos, CursorKind::Hidden)
+                } else {
+                    // Use crossterm cursor when terminal loses focus for consistent unfocues cursor behaviour
```

Comment:
```suggestion
                    // Use crossterm cursor when terminal loses focus
```

There's a little typo there in "unfocues" but I think we can drop the consistency part anyways

Reply:
Done